modules/cli-options/src/main/scala/scala/cli/commands/BloopExitOptions.scala

```diff
@@ -17,6 +17,7 @@ final case class BloopExitOptions(
 // format: on
 
 object BloopExitOptions {
-  implicit lazy val parser: Parser[BloopExitOptions] = Parser.derive
-  implicit lazy val help: Help[BloopExitOptions]     = Help.derive
+  implicit lazy val parser: Parser[BloopExitOptions]                  = Parser.derive
+  implicit lazy val parserAux: Parser.Aux[BloopExitOptions, parser.D] = parser
```

Comment:
Why do we need `Aux` here?

Reply:
I think it's because `BloopExitOptions` is being used with `@Recurse` in `UninstallOptions` (and `@Recurse` uses an implicit `Parser.Aux`)